doc/code/qml_pauli.rst

```diff
@@ -36,36 +36,35 @@ approach to represent Pauli words. A :class:`~pennylane.pauli.PauliWord` can be
 instantiated by passing a dictionary of wires and their associated Pauli operators.
 
 >>> from pennylane.pauli import PauliWord
->>> pw1 = qml.pauli.PauliWord({0:"X", 1:"Z"})  # X@Z
->>> pw2 = qml.pauli.PauliWord({0:"Y", 1:"Z"})  # Y@Z
+>>> pw1 = PauliWord({0:"X", 1:"Z"})  # X@Z
+>>> pw2 = PauliWord({0:"Y", 1:"Z"})  # Y@Z
 >>> pw1, pw2
 (X(0) @ Z(1), Y(0) @ Z(1))
 
 The purpose of this class is to efficiently compute products of Pauli words and
 obtain the matrix representation.
 
->>> pw1 * pw2
+>>> pw1 @ pw2
 (Z(0), 1j)
 >>> pw1.to_mat(wire_order=[0, 1])
 array([[ 0,  0,  1,  0],
        [ 0,  0,  0, -1],
        [ 1,  0,  0,  0],
        [ 0, -1,  0,  0]])
 
-
 The :class:`~pennylane.pauli.PauliSentence` class represents linear combinations of
 Pauli words. Using a similar dictionary based approach we can efficiently add, multiply
 and extract the matrix of operators in this representation.
 
->>> ps1 = qml.pauli.PauliSentence({pw1: 1.2, pw2: 0.5j})
->>> ps2 = qml.pauli.PauliSentence({pw1: -1.2})
+>>> ps1 = PauliSentence({pw1: 1.2, pw2: 0.5j})
+>>> ps2 = PauliSentence({pw1: -1.2})
```

Comment:
I think its actually okay to have this block explore addition and multiplication. The text I would think of to add in between sounds like null-text "We can add two Pauli words:", "We can multiply two Pauli words:", .. which is quite clear from just looking at the code I think. Do you have a concrete nicer idea?

Reply:
I agree it doesn't need any text, it's pretty self-explanatory (or already explained by the existing text) in that regard. I just meant a blank line after each output so that it renders as a few cells with white space between, so it's easier to skim.

Reply:
Ah that sounds good 👍 
✅